### ISSUE TITLE
Cannot query UIWebView by accessibilityIdentifier or accessibilityLabel

### ISSUE THREAD

**Issue by jmoody** [calabash/calabash-ios#735]
```
> query("UIWebView").first['id']
"landing page"
> query("UIWebView marked:'landing page'")
[]
> query("UIWebView").first['label']
"Zielseite"
> query("UIWebView marked:'Zielseite'")
[]
```

**Comment by jmoody** [comment]
@TobiasRoikjer and I discussed this in the context of Calabash 2.0

@krukow I think we need to drop the Web View "marked" API - it almost useless and is very confusing.

https://github.com/calabash/calabash-ios/wiki/06-WebView-Support#marked-api

**Comment by scottgoldwater** [comment]
ditto... this was almost driving me crazy! Glad I'm not the only one. I was writing a test to see if a web page is loaded and was changing the accessibility label via the UIWebView delegate.

**Comment by jmoody** [comment]
Plan is to remove for 0.18.

**Comment by jmoody** [comment]
Can't be part of 0.18.0: the release has already been cut.

**Comment by jmoody** [comment]
Available in 0.19.0.
